hazelcast/db.py

```diff
@@ -50,6 +52,7 @@ class Type(enum.Enum):
         ("null_ok", bool),
     ],
 )
+ColumnDescription.__doc__ = "ColumnDescription provides name, type and nullability information"
```

Comment:
I don't know why but it is rendered as below, which looks a bit weird

<img width="731" alt="image" src="https://user-images.githubusercontent.com/29387993/228879372-62deab17-18b4-43ac-8181-b587738db531.png">

Reply:
Fields are also rendered weirdly, but I don't know if we can override this behavior

Reply:
It's rendered the same way if I remove the doc comment, so I guess that's OK.